docs/gettingstarted/mint_tokens.md

```diff
@@ -16,6 +16,118 @@ nav_order: 9
 
 ---
 
-## Work in progress
+## Quick reference
 
-... the tokens APIs are under development
\ No newline at end of file
+- FireFly provides an abstraction layer for multiple types of tokens
+- Tokens are grouped into _pools_, which each represent a particular type or class of token
+- Each pool is classified as _fungible_ or _non-fungible_
+- In the case of _non-fungible_ tokens, the pool is subdivided into individual tokens with a unique _token index_
+- Within a pool, you may _mint_, _transfer_, and _burn_ tokens
+- Each operation can be optionally accompanied by a broadcast or private message, which will be recorded alongside the transfer on-chain
+- FireFly tracks a history of all token operations along with all current token balances
+- The blockchain backing each token connector may be the same _or_ different from the one backing FireFly message pinning
+
+## Create a pool
+
+Every application will need to create at least one token pool. At a minimum, you must always
+specify a `name` and `type` (fungible or nonfungible) for the pool.
+
+`POST` `/api/v1/namespaces/default/tokens/pools`
+
+```json
+{
+  "name": "testpool",
+  "type": "fungible"
+}
+```
+
+Other parameters:
+- You must specify a `connector` if you have configured multiple token connectors
+- You may pass through a `config` object of additional parameters, if supported by your token connector
+- You may specify a `key` understood by the connector (i.e. an Ethereum address) if you'd like to use a non-default signing identity
+
+## Mint tokens
+
+Once you have a token pool, you can mint tokens within it. With the default `firefly-tokens-erc1155` connector,
+only the creator of a pool is allowed to mint - but each connector may define its own permission model.
+
+`POST` `/api/v1/namespaces/default/tokens/mint`
```

Comment:
I think the token pool name part is missing from this path

Reply:
Okay actually I see now that it just works if there's only one pool defined. Does this URL break if there is more than one pool?

Reply:
The pool name is not in the new paths (if there is only one pool the name is optional; if there is more than one pool, you pass it in the body). The old path with the pool name is deprecated.